scm/etc/scripts/GABLS3_LSM.ncl

```diff
@@ -5,7 +5,7 @@ load "$NCARG_ROOT/lib/ncarg/nclscripts/csm/contributed.ncl"
 begin
 
 ;Define constants
-  
```

Comment:
We're still supporting NCL scripts? 😱

Reply:
Lol, I'm not sure if we officially support them. I didn't try them, only changed `Cheyenne -> Derecho` and the editor automatically cleaned up the trailing whitespace

Reply:
I've only ever run these scripts once on my local machine. They're not really meant to be run again and are saved in the repo in case there is a mistake in the forcing file and need to be regenerated. I don't think we need to "support" this to anyone but ourselves. This goes for most of the the "leftover" scripts in etc/scripts. This should probably be cleaned up at some point.